stdlib/2/typing.pyi

```diff
@@ -324,8 +324,7 @@ class IO(Iterator[AnyStr], Generic[AnyStr]):
     def __enter__(self) -> 'IO[AnyStr]': ...
     @abstractmethod
     def __exit__(self, t: Optional[Type[BaseException]], value: Optional[BaseException],
-                 # TODO: traceback should be TracebackType but that's defined in types
-                 traceback: Optional[Any]) -> bool: ...
+                 traceback: Optional[TracebackType]) -> bool: ...
```

Comment:
There's another TODO like this around L131. Should that also be fixed?

Reply:
Probably, I didn't scroll that far up. :) I'll fix it too.